src/lib/tabs/tab-truncate.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({ name: 'mdTruncate' })
+export class MdTabLabelTruncate implements PipeTransform {
+  transform(value: string, limit: number = 70, trail: String = '…'): string {
```

Comment:
Good points. I tried to be a little smarter than just doing a string length so that the label might have a better meaning if it optimized for words.

Would you like me to change the algorithm to handle longer words better or ditch this approach altogether and just do length?